projects/packages/videopress/src/client/admin/components/progress-bar/index.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+import { ProgressBarProps } from './types';
+import type React from 'react';
+
+/**
+ * Progress Bar component
+ *
+ * @param {ProgressBarProps} props - Component props.
+ * @returns {React.ReactNode} - ProgressBar react component.
+ */
+const ProgressBar: React.FC< ProgressBarProps > = ( { className, progress } ) => {
+	if ( progress == null ) {
```

Comment:
should it be `===`?

Reply:
`==` tests against `undefined` as well. It is how lodash implements `.isNil`: https://github.com/lodash/lodash/blob/master/isNil.js